.jenkins/library/vars/tests.groovy

```diff
@@ -13,32 +13,26 @@ def ACCCodeCoverageTest(String label, String compiler, String build_type) {
             timeout(globalvars.GLOBAL_TIMEOUT_MINUTES) {
                 cleanWs()
                 checkout scm
-                def cmakeArgs = helpers.CmakeArgs(build_type)
-                // Removed command "ninja code_coverage" from task for now
-                // Should re-enable it later
+                def cmakeArgs = helpers.CmakeArgs(build_type, "ON", "OFF")
                 def task = """
+                           sudo apt install -y lcov
```

Comment:
@CyanDevs Is the agreement that we setup up these dependencies on the fly instead of updating the image?

Reply:
We should install these dependencies by updating the image.